config/config.go

```diff
@@ -909,6 +909,9 @@ type StateSyncConfig struct {
 
 	// The number of concurrent chunk and block fetchers to run (default: 4).
 	Fetchers int32 `mapstructure:"fetchers"`
+
+	// Only meaningful if using P2P
+	VerifyLightBlockTimeout int `mapstructure:"verify-light-block-timeout"`
```

Comment:
why is this? I think even if we use rpc client, there's also light block verification for state sync right?

Reply:
yeah didn't notice that the rpc client has the same logic. Added